homeassistant/components/iss/config_flow.py

```diff
@@ -0,0 +1,48 @@
+"""Config flow to configure iss component."""
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_NAME, CONF_SHOW_ON_MAP
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config flow for iss component."""
+
+    VERSION = 1
+
+    async def async_step_user(self, user_input=None) -> FlowResult:
+        """Handle a flow initialized by the user."""
+        # Check if already configured
+        if self._async_current_entries():
+            return self.async_abort(reason="single_instance_allowed")
+
+        # Check if location have been defined.
+        if not self.hass.config.latitude and not self.hass.config.longitude:
```

Comment:
```suggestion
        if not self.hass.config.latitude or not self.hass.config.longitude:
```
I guess this should be `or`

Reply:
Good suggestion

Reply:
So after this suggestion I thought about it again and the [Equator](https://en.wikipedia.org/wiki/Equator#:~:text=The%20Equator%20is%20a%20circle,the%20North%20and%20South%20poles.) has a latitude of `0.0`. Although this is almost theoretical, maybe we should not prevent it.
Looking at the data in config it does not set values to `None` if they are not set, but actually set them to `0.0` this makes it hard to know if they are not set.
A longitude of `0.0` is also valid.
The combination of both 0,0 is of course invalid. Since you can't know if a user did not set one of them or it is actually 0, I suggest to revert to your original implementation, sorry for the trouble.

Reply:
That's a good point you have, didn't think much about to be honest, but glad you did. I have now reverted it 😃